vernac/prettyp.ml

```diff
@@ -821,10 +825,10 @@ let print_any_name env sigma na udecl =
   | ModuleType mp -> print_modtype mp
   | Other (obj, info) -> info.print obj
   | Undefined qid ->
-  try  (* Var locale de but, pas var de section... donc pas d'implicits *)
+  try  (* Var locale de but, pas var de section *)
```

Comment:
While you were at it you could have translated this comment (but it's not strictly necessary either).

Reply:
Done (in #18444, which this PR will inherit).